### ISSUE TITLE
FastSpeech2 convergence problem

### ISSUE THREAD

**Issue by engineergun** [TensorSpeech/TensorFlowTTS#304]
First of all thanks for this awesome repository. I have trouble with training FastSpeech2. I trained both Turkish LJSpeech formatted data and directly LJSpeech1.1. According to FastSpeech2 paper, 160k steps enough for convergence. My models were trained until 320k steps but output Mel-spectrogram and audio (decoded with Griffin-Lim) were not apprehensible. 

For LJSpeech dataset, I used directly default parameters and for duration extraction, I used tacotron2's script. I think there is a point that I missed. Can you help me with this? I am sharing 3 checkpoints.

LJSpeech1.1 40kth step mel-spect ouput.
![1](https://user-images.githubusercontent.com/67227318/95957783-d5893b80-0e08-11eb-9e13-76b611c9cc61.png)

LJSpeech1.1 160kth step mel-spect ouput.
![1](https://user-images.githubusercontent.com/67227318/95956447-f781be80-0e06-11eb-886a-7e5a7ca029f8.png)

LJSpeech1.1 320kth step mel-spect ouput.
![1](https://user-images.githubusercontent.com/67227318/95956478-04061700-0e07-11eb-991c-5ee8b2ef3c6e.png)

**Comment by dathudeptrai** [comment]
@engineergun i think there is something wrong with ur durations :)), f0 or energy :D

**Comment by ZDisket** [comment]
@engineergun I had a similar problem with a multispeaker dataset until I halved the initial learning rate, but default hparams worked well for all single-speakers. If that doesn't work, there must be something wrong with your data or how you process it.

**Comment by engineergun** [comment]
@dathudeptrai 
thank you for extremely quick response :D
My train and eval curves
![train](https://user-images.githubusercontent.com/67227318/95959808-a4f6d100-0e0b-11eb-96ed-b33fbab64474.png)
![eval](https://user-images.githubusercontent.com/67227318/95959822-aa541b80-0e0b-11eb-805b-9ba176e5cb91.png)

**Comment by dathudeptrai** [comment]
ok durations loss is wrong, it should be around 0.05 -> 0.1 for durations loss :)). @ZDisket @engineergun

**Comment by engineergun** [comment]
@dathudeptrai @ZDisket thanks i'll check it :) :+1:
